datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -110,10 +111,10 @@ select array_position(['h', 'e', 'l', 'l', 'o'], 'l', 4), array_position([1, 2,
 4 5 2
 
 # array_positions scalar function
-query III
+query error DataFusion error: SQL error: ParserError\("Expected an SQL statement, found: caused"\)
```

Comment:
these errors look not quite right -- I think there is something wrong with sqllogictest `--complete` with multi-line errors 🤔